operator_ui/src/pages/JobsIndex/JobV2Row.tsx

```diff
@@ -35,6 +36,8 @@ export const JobV2Row = withStyles(tableStyles)(({ job, classes }: Props) => {
 
   const initiator = React.useMemo(() => {
     switch (job.attributes.type) {
+      // TODO - Need to get the API to return the 'initiator' type to implement
```

Comment:
@j16r Do you return the type of DR job it is? i.e. `web`, `runlog`

Reply:
`DirectRequest     Type = "directrequest"`